internal/v5/archive_test.go

```diff
@@ -124,6 +141,21 @@ func (s *ArchiveSuite) TestGetDevelopment(c *gc.C) {
 	})
 }
 
+func (s *commonSuite) assertArchiveDownload(c *gc.C, url, id string, archiveBytes []byte) {
+	rec := httptesting.DoRequest(c, httptesting.DoRequestParams{
+		Handler: s.srv,
+		URL:     storeURL(url),
+	})
+	c.Assert(rec.Code, gc.Equals, http.StatusOK)
+	if id != "" {
```

Comment:
it's a test function.. sometimes, i guess, we do not need to check this..  (it wasn't in the original cs code)